templates/patterns/card/pattern-card--variant-search.html.twig

```diff
@@ -68,6 +68,7 @@
     'border-bottom',
     'border-md-0',
     'border-0',
+    'rounded-0',
```

Comment:
I am concerned about this because if anybody has cards with a border, we will override their styling with a class marked as important. We should already stop adding utility classes, and probably move to a pure css approach.
I'd like the feedback of the rest of the team on this though.